roachtest: reenable skipped import tests

tbg opened the pull request:
With #31875 in, import/tpch/nodes=4 has been passing reliably, and the
expectation is that the nodes=8,32 versions will follow suit (at least
they passed just now).

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/32749)
<!-- Reviewable:end -->

tbg committed:
roachtest: reenable skipped import tests

With #31875 in, import/tpch/nodes=4 has been passing reliably, and the
expectation is that the nodes=8,32 versions will follow suit (at least
they passed just now).

Release note: None

tbg commented:
Also, as of https://github.com/cockroachdb/cockroach/pull/32531, >100 pending snaps results in a Gossip alert. I'll be making the health checker strict about the alerts again (right now it just logs them but doesn't fail the test), though not in this PR.

petermattis reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> :crossed_fingers: 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/32749#-:-LSaO6tgDbifkuyxvWA0:bejbbxh)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

tbg commented:
bors r=petermattis
